src/Open3D/GUI/ListView.cpp

```diff
@@ -102,35 +101,35 @@ Widget::DrawResult ListView::Draw(const DrawContext &context) {
             ImVec2(frame.x - context.uiOffsetX, frame.y - context.uiOffsetY));
     ImGui::PushItemWidth(frame.width);
 
-    int heightNItems = int(std::floor(frame.height / ImGui::GetFrameHeight()));
+    int height_nitems = int(std::floor(frame.height / ImGui::GetFrameHeight()));
```

Comment:
is this "height_in_items"? or "items_in_frame", or "num_items_to_display" or some such?

Reply:
Apparently it is, looking at the ImGUI function...